CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Usage:
 * Fix #1935: `oc:remote-dev` goal / `ocRemoteDev` task have wrong log prefixes
 * Fix #1966: Old reference to fmp in documentation
 * Fix #1974: Remove unused methods in KubernetesResourceUtil
+* Fix #2008: Resource Validation is done before resources are generated by `k8s:resource`
```

Comment:
I'd rephrase this to
```suggestion
* Fix #2008: resources validated after their generation by `k8s:resource`
```

to explain the solution provided instead of the problem described in the issue.